### ISSUE TITLE
Argument missing

### ISSUE THREAD

**Issue by jaypatravali** [ezjong/lightprobnets#1]
Hi i am still early to the codebase but just want to get things running quickly.
if I run bash train_flyingchairs_flownetadf.sh
I get the following error.
    main()
  File "../main.py", line 73, in main
    model_and_loss = config.configure_model_and_loss(args)
  File "/home/jay/bayesian/lightprobnets/configuration.py", line 118, in configure_model_and_loss
    model = typeinf.instance_from_kwargs(args.model_class, kwargs)
  File "/home/jay/bayesian/lightprobnets/utils/type_inference.py", line 35, in instance_from_kwargs
    instance = class_constructor(**filtered_args)
  File "/home/jay/bayesian/lightprobnets/models/flownet_adf.py", line 138, in __init__
    self._flownetadf = _FlowNetADF(args, keep_variance_fn=self._keep_variance_fn)
  File "/home/jay/bayesian/lightprobnets/models/flownet_adf.py", line 56, in __init__
    self._conv1   = make_conv(   6,   64, kernel_size=7, stride=2)
  File "/home/jay/bayesian/lightprobnets/models/flownet_adf.py", line 54, in make_conv
    keep_variance_fn=keep_variance_fn)
  File "/home/jay/bayesian/lightprobnets/models/flownet_adf.py", line 22, in conv
    stride=stride, padding=pad, bias=bias, keep_variance_fn=keep_variance_fn),
  File "/home/jay/bayesian/lightprobnets/contrib/adf.py", line 152, in __init__
    False, _pair(0), groups, bias)
TypeError: __init__() takes exactly 12 arguments (11 given)

**Comment by ezjong** [comment]
Interesting, I will have a look at this on the weekend. I don't think this should be a big issue (I hope^)

**Comment by jaypatravali** [comment]
Yeah, please let us know. We are looking to add your model in our method comparison
